.cirrus.yml

```diff
@@ -1138,7 +1151,12 @@ success_task:
         CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
         TEST_ENVIRON: container
     clone_script: *noop
-    script: *noop
+    script: |
+        if [[ "$CIRRUS_CHANGE_TITLE" =~ CI:MACHINE ]] && [[ -n "$CIRRUS_PR" ]]; then
+            echo "Error: Risk of untested change merge."
+            echo "Please remove [CI:MACHINE] from title."
+            exit 1
+        fi
```

Comment:
Alternate (possibly simpler) idea: Add this as an `only_if` condition on the `Total Success` task itself.

Reply:
hrmmm, though having an actual :x: on the success task might be more helpful since it might prompt a developer to go look, and see the message.